tests/api/ClientTest.php

```diff
@@ -244,6 +244,20 @@ public function testUpdateClientInvalidMaxParallelJobs(): void
         ]);
     }
 
+    public function testUpdateClientKeepSameName(): void
+    {
+        $httpClient = $this->authenticate();
+        $client = ClientMother::base();
+        $client = $this->postClient($httpClient, $client);
+        $updateClient = ClientMother::named($client->getName());
+        $updateClientJson = $updateClient->getData();
+        $httpClient->request('PUT', $client->getIri(), ['json' => $updateClientJson]);
```

Comment:
At least change some data for the client, otherwise if the update fails but API doesn't report an error, test won't notice because PUT data is the same as POST data.